### ISSUE TITLE
Missing field `version` error when renaming variable

### ISSUE THREAD

**Issue by mikob** [autozimu/LanguageClient-neovim#252]
Missing field `version` error when trying to rename. I simply load up a python file, I can see the language server is generally working as I see new warning symbols pop up in the gutter of my python file. Then I press F4 over a simple variable to call the LanguageClient_textDocument_rename as in the example, I give the variable a new name and press enter and the error shows.

> - neovim/vim version

neovim 0.2.3-52-ga

> - Plugin version (git SHA)

4087bef (latest)

> - Minimal vimrc (e.g. <https://github.com/autozimu/LanguageClient-neovim/blob/next/min-init.vim>)
```
let g:LanguageClient_loggingLevel = 'DEBUG'
filetype plugin on
call plug#begin('~/.vim/plugged')
Plug 'autozimu/LanguageClient-neovim', {'tag': 'binary-*-x86_64*linux*'}
Plug 'junegunn/fzf', { 'dir': '~/.fzf', 'do': './install --all' }
Plug 'junegunn/fzf.vim'
call plug#end()

" language server options
let g:LanguageClient_serverCommands = {
	\ 'python': ['pyls'],
	\ }

nnoremap <silent> <f4> :call LanguageClient_textDocument_rename()<CR>
```

> - Language server name/link and version

python: https://github.com/palantir/python-language-server 0.12.1 (latest)

> - Reproduction steps (from clean state)
1. Press F4 over a variable name and you are prompted like so:
`Rename to: [var name]`
2. Press enter and error message will appear

> - Logs. 
```
2018-01-02T21:58:20.150794922+09:00 ERROR languageclient::languageclient - Error handling message. Message: {"id": 1, "jsonrpc": "2.0", "method": "textDocument/rename", "params": {"character": 10, "handle": true, "languageId": "python", "line": 108, "buftype": "", "cword": "subset", "filename": "/home/mikob/workspace/hotel-alerts/scheduler/dali.py"}}. Error: Err(ErrorImpl { code: Message("missing field `version`"), line: 0, column: 0 })
2018-01-02T21:58:20.150833339+09:00 INFO languageclient::languageclient - => {"jsonrpc":"2.0","error":{"code":-32603,"message":"missing field `version`"},"id":1}
```

**Comment by autozimu** [comment]
Thanks for reporting.

**Comment by mikob** [comment]
Thanks for the quick response. I tried the latest again, now there's a missing field `changes` error -- same scenario.

**Comment by autozimu** [comment]
Please remove this plugin and reinstall it.

vim-plug’s :PlugUpdate doesn’t work well with this repo.

Will change the recommended install approach shortly.

**Comment by mikob** [comment]
I tried the new install method mentioned in the README for bb480c it failed with something along the lines of "post-install hook failed exited with status code 127" when running `:PlugInstall` (after removing old version). So I ran the `install.sh` myself from the plugin folder -- which seemed to work. Still getting the "missing field `changes`" error though! 

```
python-language-server==0.12.1
jedi==0.11.1
```
